pkg/commands/edit/remove/removepatch.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package remove
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"log"
+	"sigs.k8s.io/kustomize/v3/pkg/commands/kustfile"
+	"sigs.k8s.io/kustomize/v3/pkg/fs"
+	"sigs.k8s.io/kustomize/v3/pkg/patch"
+	"sigs.k8s.io/kustomize/v3/pkg/pgmconfig"
```

Comment:
consider running [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) on this and the other files.

not a big deal, but we try to use that order on all the go files.

Reply:
Fixed in f11d083